solved_exercises/dani/02-tuples.py

```diff
@@ -1,14 +1,16 @@
 import itertools
 '************************************************'
 print('Check wether an element exists within a tuple')
-my_tuple = 1, 2, 3, 3, 4, 5, 6
+my_tuple = (1, 2, 3, 3, 4, 5, 6)
 value = int(input("Please enter a numeric value:\n"))
-if my_tuple.count(value)>0: print(value," is in my_tuple: ", my_tuple)
-else: print(value," isn't in my_tuple: ", my_tuple)
+if my_tuple.count(value)>0:
+    print(value," is in my_tuple: ", my_tuple)
```

Comment:
drop the tuple print out

Reply:
:), I'm running complete py file and I use it more as debug ... can be deleted